python/pyspark/ml/classification.py

```diff
@@ -679,20 +682,20 @@ class LinearSVC(_JavaClassifier, _LinearSVCParams, JavaMLWritable, JavaMLReadabl
     def __init__(
```

Comment:
If class uses `@keyword_only` decorators, you'll have to add 

```python
_input_kwargs: Dict[str, Any]
```
at the beginning of the class body.